Merge the staff handbook (#571)
publiccodenet/about: activities/staff-management/parental-leave.md

---
type: Guide

> explains: how parental leave works. links to the government's website for detailed information

Reviewer: This should be one sentence, following the explains logically

```suggestion
explains: what to do and what to know when you or your partner are expecting
```

---

# Maternity and paternity leave

Pregnant employees are entitled to 4-6 weeks pregnancy leave 
(before the due date) and at least 10 weeks maternity leave (after childbirth). 

### Paternity leave

You are entitled to 1 week of partner or paternity leave after your partner gives birth.

### More information

[More information about maternity and partner / paternity leave can be found here:](https://business.gov.nl/regulation/leave-schemes/)

> If you have questions about parental leave, please reach out to HR <hr@publiccode.net>.

Reviewer: What I'm missing here, and what should be leading is 'how do you do it'. As a guide it should set out what you need to do if you or your partner is pregnant.

---
type: Guide
explains: what to do and what to know when you or your partner are expecting
---

> # Maternity and paternity leave

Reviewer 2: ```suggestion
# Maternity, partner/paternity and parental leave
```

> 

Reviewer 2: ## Maternity leave

Pregnant employees are entitled to 4-6 weeks pregnancy leave (before the due date)

> and at least 10 weeks maternity leave (after childbirth).

Reviewer: As per the previous suggestion

```suggestion
and at least 10 weeks birth leave.
```

> ## Parental leave

Reviewer 2: ```suggestion
## Partner/paternity leave
```

> ## Paternity leave

Author: Done!

> You are entitled to 1 week of parental leave after your partner gives birth.

Reviewer 2: ```suggestion
You are entitled to 1 week of partner/paternity leave after your partner gives birth.
```

### How to

> If you are pregnant you have to let HR know at least 3 weeks before you want to go on maternity

Reviewer: ```suggestion
If you are pregnant you have to let HR know at least 3 weeks before you want to go on pregnancy
```

leave. But, the sooner you report it, the better, leaving enough time to make any

> agreements about your absence, and plan a handover of your tasks. For paternity leave, let us know

Reviewer: ```suggestion
agreements about your absence, and plan a handover of your tasks. For partner leave, let us know
```

when your partner has given birth by 9am the next working day.

#### More information

> [More information about maternity and partner / paternity leave can be found here:](https://business.gov.nl/regulation/leave-schemes/)

Reviewer 2: ```suggestion
[Read more about maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) (on the Dutch government website).
```

> [Read more about maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) (on the Dutch government website).

Reviewer: ```suggestion
* [Zwangerschapsverlof en bevallingsverlof (Dutch)](https://www.rijksoverheid.nl/onderwerpen/zwangerschapsverlof-en-bevallingsverlof) on rijksoverheid.nl
* [Maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) on business.gov.nl.
```

---
type: Guide
explains: what to do and what to know when you or your partner are expecting
---

# Pregnancy, birth, and partner leave

Pregnant employees are entitled to 4-6 weeks pregnancy leave (before the due date)
and at least 10 weeks birth leave.

## Partner leave

You are entitled to 1 week of partner leave after your partner gives birth.

### How to

If you are pregnant you have to let HR know at least 3 weeks before you want to go on pregnancy
leave. But, the sooner you report it, the better, leaving enough time to make any
agreements about your absence, and plan a handover of your tasks. For partner leave, let us know
when your partner has given birth by 9am the next working day.

#### More information

* [Zwangerschapsverlof en bevallingsverlof (Dutch)](https://www.rijksoverheid.nl/onderwerpen/zwangerschapsverlof-en-bevallingsverlof) on rijksoverheid.nl
* [Maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) on business.gov.nl.

> If you have questions about parental leave, or need to take parental leave, please reach out to HR <hr@publiccode.net>

Reviewer 2: ```suggestion
If you have questions about parental leave, or need to take parental leave, please reach out to HR <hr@publiccode.net>.
```
